js/react.md

```diff
@@ -238,26 +220,27 @@ const signUpConfig = {
       type: 'string'
     },
     {
-      label: 'Custom Attribute',
-      key: 'custom_attr',
-      required: false,
+      label: 'Email',
+      key: 'email',
+      required: true,
       displayOrder: 4,
-      type: 'string',
-      custom: true
+      type: 'string'
     }
   ]
 };
 const usernameAttributes = 'My user name';
 
-export default withAuthenticator(App, { 
-  signUpConfig, 
+export default withAuthenticator(App, {
+  signUpConfig,
   usernameAttributes
 });
 ```
```

Comment:
Using custom attributes here needs additional configuration in aws cognito or at initial setup. For some one following the tutorial, it is misleading that this would work and results in error.

Reply:
Got it. Thanks for updating this.